tests/test_arrange.py

```diff
@@ -68,10 +84,10 @@ def test_PrepareDcm2niixInput_Zip_Tar_Archive_MatchValidDataset(tmpdir, version,
     old_dir = f"{ASSETS_DIR}/valid_dataset/{version}" # corresponding unzipped dir
 
     # define and populate new dir
-    if os.path.exists(f"{ASSETS_DIR}/tmp"):
-        shutil.rmtree(f"{ASSETS_DIR}/tmp")
-    os.mkdir(f'{ASSETS_DIR}/tmp')
-    new_dir = arrange.prepare_dcm2niix_input(infile, False, f"{ASSETS_DIR}/tmp")
+    if os.path.exists(tmpdir):
+        shutil.rmtree(tmpdir)
+    os.mkdir(tmpdir)
```

Comment:
You can drop this 3 lines. The tmpdir fixture from pytest is going to do that for you. That's the beauty of it :) Free up your mind.

Reply:
Oh yes, missed this. Quite beautiful! :)